samples/excel/tables/create-table.yaml

```diff
@@ -0,0 +1,94 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Create a table
+description: Creates a table with four columns and seven rows.
+script:
+  content: |+
+    $('#create-table').click(createTable);
+
+    /* Create a new table with some sample data */
+    function createTable() {
+        Excel.run(async (context) => {
+            let sheet: Excel.Worksheet;
+            try {
+                // Queue a command to add a new sheet
+                sheet = context.workbook.worksheets.add();
+
+                // Call the helper to create the table
+                await queueCreateTable(sheet);
+
+                //Queue a command to activate the sheet
+                sheet.activate();
+
+                // Run all of the queued up commands
+                await context.sync();
+            }
+            catch (error) {
+                OfficeHelpers.Utilities.log(error);
+            }
+        })
+    }
+
+    /* Helper to create a new table with some sample data */
+    async function queueCreateTable(sheet: Excel.Worksheet) {
+
+            // Queue a command to add a table to the sheet
+            let expensesTable = sheet.tables.add('A1:D1', true);
+
+            // Queue a command to set the header row values
+            expensesTable.getHeaderRowRange().values = [["DATE", "MERCHANT", "CATEGORY", "AMOUNT"]];
+
+            // Queue a command to add rows to the table
+            expensesTable.rows.add(null, [
```

Comment:
Should we use title casing in sample data rather than all upper case? It seems friendlier to me.